prestoadmin/util/slider.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Utility functions for dealing with slider
+"""
+
+
+def degarbage_json(garbage_output):
+    nest_level = 0
```

Comment:
The other possibility, by the way, is to use Slider's -out option which can be used to get Slider to output clean JSON to a file. That option circumvents the need to try to pluck some JSON out of a string, but comes with other problems.

Notably, we need to be able to write a file to a sufficiently unique path that nothing blows up if multiple people are getting the status at the same time. Then we need to clean it up, which is simple enough if everything is going well, but risks cluttering up /tmp if pa somehow crashes between having slider write the file and deleting it. It goes without saying that the only thing we should be doing in that window is trying to grab the file via fabric, but Murphy's law and all that.